test/tscc/lib

```diff
@@ -0,0 +1 @@
+../../lib
```

Comment:
I needed to add a symlink so that we both get a fresh project root _and_ not have generated closure namespaces with `..` in them.